public/style.css

```diff
@@ -29,6 +29,11 @@ time {
   color: #393;
 }
 
+.with_images {
+  float: right;
+  margin-left: 5px;
+}
+
```

Comment:
The main reason I added this was to make it remain on the same line as the column view checkbox, so that it was clear that "with images" only applied to that view.  If I change it to be the default, this goes away.